README.md

```diff
@@ -29,3 +31,22 @@ setRedListToken('<your-token>')
 ```
 
 Be sure to reference the [Red List API docs](http://apiv3.iucnredlist.org/api/v3/docs)
+
+## Threat - Species threat information
+* [Threat.fetch()](#threat-fetch)
+
+<h3 id='threat-fetch'>Threat.fetch(options)
+
+Retrieves a list of threats by either species `name` or species `id`
+
+##### `options` (required) - **[Object]**
+* `name` - **[String]** Name of species to filter threats by
+* `id` - **[String]** Id of species to filter threats by
```

Comment:
"If `id` is not provided, `name` must be provided. If both `id` and `name` are provided, `id` will take precendent and `name` will be ignored"